group_3/script.R

```diff
@@ -0,0 +1,122 @@
+# Group 3
+# Matthew Bodziony, Daniel Choi, Cole Groeneveld
+
+# Path to Git Bash on your computer
+git_bash_path <- "C:/Program Files/Git/git-bash.exe"
```

Comment:
This is fine but think about what if I have the Git Bash executable installed on another location or if I have Mac?